src/cmd/run.rs

```diff
@@ -51,6 +56,14 @@ fn ethereum_rpc() -> EthereumRPC {
         .unwrap_or_else(|_| {
             if env::var("ETHEREUM_ROPSTEN").is_ok() {
                 crate::conf::DEFAULT_ETHEREUM_ROPSTEN_RPC.into()
+            } else if env::var("HECO_MAINNET").is_ok() {
+                crate::conf::DEFAULT_HECO_MAINNET_RPC.into()
+            } else if env::var("HECO_TESTNET").is_ok() {
+                crate::conf::DEFAULT_HECO_TESTNET_RPC.into()
+            } else if env::var("BSC_MAINNET").is_ok() {
+                crate::conf::DEFAULT_BSC_MAINNET_RPC.into()
+            } else if env::var("BSC_TESTNET").is_ok() {
+                crate::conf::DEFAULT_BSC_TESTNET_RPC.into()
             } else {
```

Comment:
for different networks, if we start separate shadow instances, why we don't use a single environment variable such as `ETHEREUM_RPC`.

Reply:
Yes, a single env variable is enough. i followed the old convention `ETHEREUM_RPC `, `ETHEREUM_ROPSTEN` to add these new envs

Reply:
When we start several different shadow instance in the same host, that would be necessary to distinguish different instance by envs

Reply:
if the env is global. the `ETHEREUM_RPC` env will prevent the others. So it may be better to use a local env for each instance.

Reply:
I'm wrong about this i think. It's not possible to start different instance in same host by different envs

Reply:
Yes, that's the only way